third_party/BLAKE2/README-Toit.txt

```diff
@@ -0,0 +1,2 @@
+From https://github.com/BLAKE2/BLAKE2
```

Comment:
Add a version number number and/or commit hash.

We/I usually call these files "README.toitware".
See:
src/compiler/third_party/libyaml/README.toitware
src/compiler/third_party/nlohmann/README.toitware
src/compiler/third_party/semver/README.toitware
src/compiler/third_party/termcolor-c/README.toitware
third_party/rpi/README.toitware

They usually have a bit more structure:
For example:
```
Name: rpi
Short Name: rpi
URL: https://github.com/mmozeiko/rpi
Version: -
Revision: 4756900efa83f1cefd60e01a20b04f84b0626e0b
Date: 2022-07-06
License: Unlicense license

Description:
...
```

Off topic: we should create third-party-XX forks into toitlang/toitware for every third-party we use. In case the original disappears.